packages/api/src/external/canvas/canvas-event-listener.ts

```diff
@@ -0,0 +1,85 @@
+import CanvasSDK from "@metriport/core/external/canvas/index";
+import { createFullNote } from "@metriport/core/external/canvas/note";
+import { out } from "@metriport/core/util/log";
+import { getEnvVarOrFail } from "@metriport/shared";
+import {
+  PatientEvents,
+  patientEvents,
+  CanvasIntegrationEvent,
+} from "../../event/medical/patient-event";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+import { CONVERSION_WEBHOOK_TYPE } from "../../command/medical/document/process-doc-query-webhook";
+import { Config } from "../../shared/config";
+
+const { log } = out("CANVAS EVENT LISTENER");
+
+function isCanvasIntegrationEvent(event: CanvasIntegrationEvent): boolean {
+  return (
+    !Config.isProdEnv() &&
```

Comment:
nit: I think enabled/disabled should not be part of `isCanvasIntegrationEvent`; unless we want to call the function something else (I'd have the check for enabled/disabled right on the listener function, or have a small function called `isCanvasIntegrationEnabled()`